### PR TITLE
Add kubernetes local deployment

### PR DESCRIPTION
**Pull request by YuryHrytsuk**
## What do these changes do?

Add local deployment for kubernetes cluster

FYI: @GitHK @pcrespov @matusdrobuliak66 

## Related issue/s
* closes https://github.com/ITISFoundation/osparc-ops-environments/issues/696

## Related PR/s
* https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/merge_requests/644

## Checklist
- [X] I tested and it works

### CODE REVIEW

File: charts/Makefile [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+include ${REPO_BASE_DIR}/scripts/common.Makefile
+include $(REPO_CONFIG_LOCATION)
+
+CONFIG_DIR := $(shell dirname $(REPO_CONFIG_LOCATION))
+CHART_DIRS := $(wildcard $(REPO_BASE_DIR)/charts/*/)
+
+.PHONY: .check-helmfile-installed
+.check-helmfile-installed:
+	@if ! command -v helmfile >/dev/null 2>&1; then \
+			echo "'helmfile' is not installed. Install it to continue ...";\
```
Comment (sanderegg) [inline charts/Makefile:12]: you might want to add in the README how to install all these requirements?

**Review by sanderegg — APPROVED** [review]
nice!
maybe you could add in a README file how to install all these new requirements?

File: charts/adminer/values.yaml.gotmpl [hunk 2]
```diff
@@ -49,21 +49,19 @@ ingress:
   enabled: true
   className: ""
   annotations:
-    cert-manager.io/cluster-issuer: "letsencrypt-production"
+    cert-manager.io/cluster-issuer: "cert-issuer"
   tls:
     - hosts:
-        - k8s.monitoring.${MACHINE_FQDN}
+        - k8s.monitoring.{{ requiredEnv "MACHINE_FQDN" }}
```
Comment (pcrespov) [inline charts/adminer/values.yaml.gotmpl:55]: Q: what is the advantage of gotmpl vs jinja2?

File: scripts/create_local_k8s_cluster.bash [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        exit 0
+fi
+
+if [[ ! -f "$KIND_CONFIG_FILE" ]]; then
+    echo "Error: Configuration file $KIND_CONFIG_FILE not found."
+    exit 1
+fi
+
+# create a k8s cluster
+kind create cluster --config "$KIND_CONFIG_FILE" --name "$KIND_CLUSTER_NAME"
```
Comment (pcrespov) [inline scripts/create_local_k8s_cluster.bash:32]: kindly create ... very polite CLI command :-)

**Review by pcrespov — APPROVED** [review]
thx

File: charts/adminer/values.yaml.gotmpl (hunk 2 above)
Comment (YuryHrytsuk) [reply charts/adminer/values.yaml.gotmpl:55]: we use `helmfile`. `helmfile` natively supports `values.yaml.gotmpl`. If we used `jinja2`, then we would have to manually run it.

File: charts/Makefile (hunk 1 above)
Comment (YuryHrytsuk) [reply charts/Makefile:12]: done

**Pull request merged**